annotations/src/main/java/com/statful/client/annotations/Timer.java

```diff
@@ -38,4 +38,9 @@
      * @return An array of aggregations to send to Statful
      */
     Aggregation[] aggregations() default {};
+
+    /**
+     * @return An int of the sampleRate to send to Statful
+     */
+    int sampleRate() default 0;
```

Comment:
If not specified it sends the one defined in the Statful Client. I think making the default 100 can make the user think it will send all the metrics when in reality, the sample rate can be another number in the client.